packages/boot/tools/supports.ts

```diff
@@ -129,7 +130,14 @@ export const getNodeTestVaultsConfig = async (
     );
   }
 
-  const testConfigPath = `${bundleDir}/${basename(specifier)}`;
+  const configFilenameParts = [
+    'config',
+    discriminator,
+    new Date().toISOString().replaceAll(/[^0-9TZ]/g, ''),
+    String(Math.random()).replace(/^[^.]*[.]/, ''),
```

Comment:
I would prefer a fixed-width random component. If I understand correctly, this can produce quite a variable range of possible lengths of digits.

Reply:
Done; this part will be 8 decimal digits.